internal/cmd/server_test.go

```diff
@@ -6,12 +6,52 @@
 package cmd
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	invalidGatewayConfig = `
+kind: EnvoyGateway
+apiVersion: config.gateway.envoyproxy.io/v1alpha1
+gateway: {}
+`
+)
+
 func TestGetServerCommand(t *testing.T) {
 	got := getServerCommand()
 	assert.Equal(t, "server", got.Use)
 }
+
+func TestGetConfigValidate(t *testing.T) {
+	tests := []struct {
+		name   string
+		input  string
+		errors []string
+	}{
+		{
+			name:   "invalid gateway",
```

Comment:
can you also add a good case ?
here's an example
https://github.com/envoyproxy/gateway/blob/main/internal/provider/kubernetes/config/envoy-gateway/envoy-gateway.yaml

Reply:
Done. Thanks for reviewing!